<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_NewSupplier</class>
 <widget class="QWidget" name="Form_NewSupplier">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>258</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>258</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Fournisseur</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>221</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_Nom"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Adresse</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_Adresse"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Code ZIP</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Pays</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_ZIP"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_Country"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Nom resp.</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_RespName"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Téléphone</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEdit_telephone"/>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="lineEdit_email"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_AjouterFournisseur">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajouter fournisseur</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ajouter un fournisseur</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
